frappe/website/doctype/web_form/web_form.py

```diff
@@ -368,12 +368,14 @@ def has_web_form_permission(self, doctype, name, ptype='read'):
 
 
 @frappe.whitelist(allow_guest=True)
-def accept(web_form, data, docname=None, for_payment=False):
+def accept(web_form, data, docname=None, files=None, for_payment=False):
 	'''Save the web form'''
 	data = frappe._dict(json.loads(data))
+	files = json.loads(files) if files else []
```

Comment:
We don't have to send the files explicitly again since the docname is already attached to the file doc here and it will be linked: https://github.com/frappe/frappe/pull/11734/files#diff-be315ceb896313aa775a3bfc50100cd8a4376f7cc0db9d2827558c9a8b373780

This however will not be true for new doctypes, in that case the Doctype can be filled for the file in `accept method`